cognite/neat/workflows/steps/lib/graph_loader.py

```diff
@@ -321,10 +321,10 @@ class GenerateCDFRelationshipsFromGraph(Step[CategorizedRelationships]):
 
     def run(  # type: ignore[override]
         self, rules: RulesData, cdf_client: CogniteClient, solution_graph: SolutionGraph
-    ) -> (FlowMessage, CategorizedRelationships):
+    ) -> tuple[FlowMessage, CategorizedRelationships]:
         # create, categorize and upload relationships
         rdf_relationships = rdf2relationships(
-            solution_graph.graph.get_graph(),
+            solution_graph.graph,
```

Comment:
Bugfix?  I considered doing this in my PR, but I did not want to change behavior when I only did `mypy`

Reply:
solution_graph.graph -> provides graph_store, and that is what `rdf2relationships` expects as input